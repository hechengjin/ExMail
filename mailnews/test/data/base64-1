Content-Type: text/plain; charset=iso-8859-1
Content-Transfer-Encoding: base64

DQpIZWxsbywgd29ybGQhIChBZ2Fpbi4uLikNCg0KTGV0J3Mgc2VlIGhvdyB3ZWxsIGJhc2U2NCB0ZXh
0IGlzIGhhbmRsZWQuICAgICAgICAgICAgICAgICAgICAgICAgICAgIFlheSwgbG90cyBvZiBzcGFjZX
MhIFRoZXJlJ3MgZXZlbiBhIENSTEYgYXQgdGhlIGVuZCBhbmQgb25lIGF0IHRoZSBiZWdpbm5pbmcsI
GJ1dCB0aGUgb3V0cHV0IHNob3VsZG4ndCBoYXZlIGl0Lg0K
