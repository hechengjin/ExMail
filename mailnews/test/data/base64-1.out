Hello, world! (Again...)

Let's see how well base64 text is handled.                            Yay, lots of spaces! There's even a CRLF at the end and one at the beginning, but the output shouldn't have it.